Sources/Alloy/Encoders/Normalize.swift

```diff
@@ -0,0 +1,86 @@
+import Metal
+
+final public class Normalize {
```

Comment:
I believe this naming is slightly inconsistent with the rest of codebase, so can we please rename it to `Normalization`